v4/source/usage.yaml

```
@@ -24,3 +24,29 @@
           $ref: "#/components/responses/Unauthorized"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /usage/storage:
+    get:
+      tags:
+        - usage
+      summary: Get the total file storage usage for the instance in bytes.
+      description: >
+        Get the total file storage usage for the instance in bytes rounded down to the most significant digit.
+        Example: returns 4000 instead of 4321
+
+        ##### Permissions
+
+        Must be authenticated.
+
+        __Minimum server version__: 7.1
```

I think it should be 7.0
```suggestion
        __Minimum server version__: 7.0
```

---

The v7.0 milestone was removed before that PR was merged. We were way pass the deadline for v7.0. T
he PR wont be cherry-picked onto the v7.0 release branch, but it will definitely be included in the v7.1 release. 

The API wasn't required for v7.0. It was required for cloud, but our GH milestones don't include cloud releases.